Tools/WinMLRunner/OutputHelper.h

```diff
@@ -265,98 +318,66 @@ class OutputHelper
+                     << "Bind (ms)" << ","
+                     << "Evaluate (ms)" << ","
+                     << "Total Time (ms)" << ","
+                     << "Working Set Memory usage (evaluate) (MB)" << ","
+                     << "GPU Dedicated memory usage (evaluate) (MB)" << ","
+                     << "GPU Shared memory usage (evaluate) (MB)" << ","
+                     << "Wall-clock Load (ms)" << ","
+                     << "Wall-clock Bind (ms)" << ","
+                     << "Wall-clock Evaluate (ms)" << ","
+                     << "Wall-clock total time (ms)" << std::endl;
             }
 
-            fout << modelName << "," << args.NumIterations() << ",";
+            fout << modelName << ","
+                 << modelBinding << ","
+                 << inputBinding << ","
+                 << inputType << ","
+                 << numIterations << ","
+                 << firstRunIgnored << ","
+                 << (isnan(loadTime) ? "N/A" : std::to_string(loadTime)) << ","
+                 << bindTime << ","
+                 << evalTime << ","
+                 << loadTime + bindTime + evalTime << ","
+                 << (isnan(evalMemoryUsage) ? "N/A" : std::to_string(evalMemoryUsage)) << ","
+                 << (isnan(gpuEvalDedicatedMemoryUsage) ? "N/A" : std::to_string(gpuEvalDedicatedMemoryUsage)) << ","
+                 << (isnan(gpuEvalSharedMemoryUsage) ? "N/A" : std::to_string(gpuEvalSharedMemoryUsage)) << ","
+                 << m_clockLoadTime << ","
+                 << clockBindTime << ","
+                 << clockEvalTime << ","
+                 << m_clockLoadTime + clockBindTime + clockEvalTime << std::endl;
 
-            if (args.UseCPUandGPU() || args.UseCPU())
-            {
-                fout << m_CPULoadTime << ","
-                << m_CPUBindTime << ","
-                << m_CPUEvalTime << ","
-                << m_CPULoadTime + m_CPUBindTime + m_CPUEvalTime << ","
-                << m_CPUEvalMemoryUsage << ",";
-            }
-            if (args.UseCPUandGPU() || args.UseGPU())
-            {
-                fout << "N/A" << ","
-                << m_GPUBindTime << ","
-                << m_GPUEvalTime << ","
-                << m_GPUBindTime + m_GPUEvalTime << ","
-                << m_GPUEvalDedicatedMemoryUsage << ","
-                << m_GPUEvalSharedMemoryUsage;
-            }
-
-            fout << m_clockLoadTime << ","
-            << m_clockBindTime << ","
-            << m_clockEvalTime << ","
-            << m_clockLoadTime + m_clockBindTime + m_clockEvalTime << ","
-            << std::endl;
             fout.close();
         }
     }
     
     void Reset() 
     {
-         m_GPUBindTime = 0;
-         m_GPUEvalTime = 0;
-         m_GPUEvalSharedMemoryUsage = 0;
-         m_GPUEvalDedicatedMemoryUsage = 0;
-
-         m_CPULoadTime = 0;
-         m_CPUBindTime = 0;
-         m_CPUEvalTime = 0;
-         m_CPUEvalMemoryUsage = 0;
-
-
+         m_clockEvalTime = 0;
```

Comment:
Ideally, we shouldn't keep any member variable in the output helper (especially when we modify them in the Print methods), but this is a good start.